gptel-transient.el

```diff
@@ -185,22 +185,18 @@ which see."
   (transient-parse-suffixes
    'gptel-system-prompt
    (cl-loop for (type . prompt) in gptel-directives
-       with taken
+       ;; Avoid clashes with the custom directive key
+       with unused-keys = (remove ?h (number-sequence ?a ?z))
```

Comment:
Thanks for catching this case! I forgot about `h` being taken.

Reply:
Could you use `delete` instead of `remove` here since `remove` just does `(delete elt (copy-sequence seq))`?  We don't need to generate a copy of the sequence.

Reply:
Done